### ISSUE TITLE
caddy-docker-proxy as file server ?

### ISSUE THREAD

**Issue by accforgithubtest** [lucaslorentz/caddy-docker-proxy#636]
Thank you for this neat project, makes it really easy to set up reverse proxy straight in the docker compose files.

Is it possible to use an existing caddy-docker-proxy reverse proxy as a file server too ? 
Assuming I have a bunch of photos and videos in a folder /camera, can I somehow mount this folder and make caddy-docker proxy be both a reverse proxy and a file server ? 

[I am not a developer but I learnt docker for self-hosting my home server, so apologies in advance if you have to eli5]

**Comment by francislavoie** [comment]
Yes you can. Just add the labels for your file_server to the CDP container itself (as well as the volume mount of course).

**Comment by lucaslorentz** [comment]
Yes, you can use caddy-docker-proxy as a file server using labels:

```yaml
labels:
  caddy: files.example.com
  caddy.file_server: ""
  caddy.root: "* /srv"
```

Mount your photos folder into the caddy container at `/srv` (or any path you choose) and configure the labels accordingly.
